### PR TITLE
Added methods to quickly reply to a message with Markdown or HTML

### PR DESCRIPTION
**Pull request by JosXa**

### CODE REVIEW

File: telegram/message.py [hunk 1]
```diff
@@ -305,6 +306,40 @@ def reply_text(self, *args, **kwargs):
         self._quote(kwargs)
         return self.bot.send_message(self.chat_id, *args, **kwargs)
 
+    def reply_markdown(self, *args, **kwargs):
+        """
+        Shortcut for ``bot.sendMessage(update.message.chat_id, parse_mode=ParseMode.MARKDOWN, *args, **kwargs)``
```
Comment (Eldinnie) [inline telegram/message.py]: Please change to:
```
Shortcut for::

<two time indent>bot.sendMessage(update.message.chat_id, parse_mode=ParseMode.MARKDOWN, *args, **kwargs)
```

File: telegram/message.py [hunk 2]
```diff
@@ -305,6 +306,40 @@ def reply_text(self, *args, **kwargs):
...
+        Sends a message with markdown formatting.
+
+        Keyword Args:
```
Comment (Eldinnie) [inline telegram/message.py]: Args: (without keyword)

File: telegram/message.py [hunk 3]
```diff
@@ -305,6 +306,40 @@ def reply_text(self, *args, **kwargs):
...
+            quote (Optional[bool]): If set to ``True``, the message is sent as an actual reply to
+                this message. If ``reply_to_message_id`` is passed in ``kwargs``, this parameter
```
Comment (Eldinnie) [inline telegram/message.py]: Indent continuing line with two indents.

**Comment by rahiel** [comment]
Could you change `bot.sendMessage` to `bot.send_message`, and `bot.editMessageText` to `bot.edit_message_text` in the docstrings? We prefer snake_case method names now.

**Comment by JosXa** [comment]
Closed by #827.

**Pull request closed without merging**